src/Sentry/TransactionTracer.cs

```diff
@@ -234,6 +240,8 @@ public TransactionTracer(IHub hub, ITransactionContext context)
             _instrumenter = transactionContext.Instrumenter;
         }
 
+        var idleTimeout = idleTimeoutOverride ?? hub.GetSentryOptions()?.IdleTimeout;
```

Comment:
```suggestion
```
When the transaction gets created and there is no idle timeout the transaction does not time out.

Reply:
@bitsandfoxes, isn't this problematic? If a `SentryOptions` has an IdleTimeout specified, it will be ignored as with this change we no longer take it into consideration.